### ISSUE TITLE
Screensaver doesn't start after updating to 3.2

### ISSUE THREAD

**Issue by zquestz** [linuxmint/Cinnamon#5920]
TLDR -> Regression in https://github.com/linuxmint/cinnamon-desktop/commit/4c629a090903c8328bc64c7dbda144012486d60a

After updating all cinnamon packages in arch, I got an error starting my screensaver (cinnamon-screensaver 3.0.1).

```
 $ cinnamon-screensaver

(cinnamon-screensaver:4540): GLib-GIO-ERROR **: Settings schema 'org.cinnamon.desktop.screensaver' does not contain a key named 'logout-enabled'
[1]    4540 trace trap (core dumped)  cinnamon-screensaver
```

In order to fix the issue I had to add the following to `/usr/share/glib-2.0/schemas/org.cinnamon.desktop.screensaver.gschema.xml` and update the schemas with `sudo glib-compile-schemas .`:

```
    <key type="b" name="logout-enabled">
      <default>false</default>
      <summary>Logout enabled</summary>
      <description>Set this to TRUE to allow logout.</description>
    </key>
    <key type="s" name="logout-command">
      <default>"cinnamon-session-quit-force-noprompt"</default>
      <summary>Logout command</summary>
      <description>
        The command executed when logging out.
      </description>
    </key>
    <key type="u" name="logout-delay">
      <default>0</default>
      <summary>Logout delay</summary>
      <description>The number of seconds before showing the logout button.</description>
    </key>
```

Is this expected? Is something else going on I am not understanding properly? Perhaps just an issue with the archlinux package?

Seems someone else had a simliar issue in 2011 - https://www.linuxquestions.org/questions/showthread.php?p=5628367#post5628367

**Comment by JosephMcc** [comment]
https://github.com/linuxmint/Cinnamon/issues/5916
